src/components/organisms/tables/TranslationMemoriesTable/TranslationMemoriesTable.tsx

```diff
@@ -130,7 +126,7 @@ const TranslationMemoriesTable: FC = () => {
           <div className={classes.tagsRow}>
             {map(getValue(), (value) => {
               const tagName = find(tagsOptions, { value })?.label || ''
-              return <Tag label={tagName} value key={value} />
+              return !!tagName && <Tag label={tagName} value key={value} />
```

Comment:
Maybe {tagName ? <Tag label={tagName} value key={value} /> : null} can be better, since that is the standard doc recommendation